src/governance-object.cpp

```diff
@@ -592,7 +592,9 @@ bool CGovernanceObject::IsCollateralValid(std::string& strError, bool& fMissingC
     }
 
     if(nConfirmationsIn < GOVERNANCE_FEE_CONFIRMATIONS) {
-        strError = strprintf("Collateral requires at least %d confirmations - %d confirmations", GOVERNANCE_FEE_CONFIRMATIONS, nConfirmationsIn);
+        strError = strprintf("Collateral requires at least %d confirmation%s - %d confirmations",
```

Comment:
User can submit proposal with 1 confirmation, but actually it will be processed when 6 confirmations have passed. Perhaps we need another message for preliminarily accepted proposals?